AnkiDroid/src/main/java/com/ichi2/anki/StudyOptionsFragment.kt

```diff
@@ -608,11 +609,18 @@ class StudyOptionsFragment :
             }
 
             // Set deck description
+            @Language("HTML")
             val desc: String =
                 if (isDynamic) {
                     resources.getString(R.string.dyn_deck_desc)
                 } else {
-                    col.decks.current().description
+                    col.decks.current().let { deck ->
+                        if (deck.markdownDescription) {
+                            renderMarkdown(deck.description, sanitize = true)
+                        } else {
+                            deck.description
+                        }
+                    }
```

Comment:
nit: an option to reduce indentation

```suggestion
                    val deck = col.decks.current()
                    if (deck.markdownDescription) {
                        renderMarkdown(deck.description, sanitize = true)
                    } else {
                        deck.description
                    }
```